### DISCUSSION TITLE
Plate solve using phone camera to update position on UI

### DISCUSSION THREAD

**Discussion by TacoTakumi** [Stellarium/stellarium#3369] [Ideas]
I think this should be doable and would make for a killer feature.

I would love to mount my phone to my dobsonian telescope and use plate solving from astrometry.net https://nova.astrometry.net/upload to update the position on screen.

Is this possible with such a wide view?  Essentially this would turn the phone into a smart guide camera and very useful for push-to.  I think it could also be used with the telescope control feature to turn it into a goto system as well.

I found this related older discussion: #2091 

Thoughts?

**Comment by gzotti** [comment]
You can write independent software to plate-solve your smartphone screen and then just set Stellarium's position via RemoteControl API. A sensor frame configured in Stellarium's Oculars plugin would then show the camera's frame.
